modules/Utilities.js

```diff
@@ -686,8 +686,9 @@ class Utilities {
     }
 
     static generateRsvSignature(message, web3, privateKey) {
+        const sortedMessage = JSON.stringify(Utilities.sortObject(message));
```

Comment:
Since this function is used in a lot of places, could you add a check whether `message` is already a string so that we don't accidentally break anything?